src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -177,7 +177,7 @@ function MapViz(props: VisualizationProps<Options>) {
 
   const {
     markers,
-    totalEntityCount,
+    totalEntitiesCount: totalEntityCount,
```

Comment:
I'm not sure we really want to start renaming things that affect other components, such as `MapVisualization`.
Sorry if I gave that impression in [this comment](https://github.com/VEuPathDB/web-eda/pull/1656#discussion_r1124553122).

I was really just referring to the property names of `SemiTransparentHeader` and I should have been clearer that the inclusion of "Sample" in those names was my main objection.

Reply:
👍 My bad. I'll revert those.